subsys/net/lib/sntp/sntp.c

```diff
@@ -76,7 +92,43 @@ static int32_t parse_response(uint8_t *data, uint16_t len, uint32_t orig_ts,
 		return -EINVAL;
 	}
 
-	time->fraction = ntohl(pkt->tx_tm_f);
+#if IS_ENABLED(CONFIG_SNTP_UNCERTAINTY)
```

Comment:
Using `IS_ENABLED()` does not give any benefit here so we have been using normal
```
#if defined(CONFIG_SNTP_UNCERTAINTY)
```
in other parts of the code space in zephyr. Please change to use `if defined()` instead.